love/src/Config.js

```diff
@@ -436,6 +436,92 @@ export const severityEnum = {
   critical: 4,
 };
 
+// MT Camera Hexapod
+export const hexapodStatusStatetoStyle = {
+  UNKNOWN: 'undefined',
+  ENABLED: 'enabled',
+  DISABLED: 'disabled',
+  FAULT: 'fault',
+  OFFLINE: 'offline',
+  STANDBY: 'standby',
+};
+
+export const hexapodCommandableByDDSStateMap = {
+  false: 'NOT COMMANDABLE',
+  true: 'COMMANDABLE',
+};
+
+export const hexapodCommandableByDDSStatetoStyle = {
+  'NOT COMMANDABLE': 'alert',
+  COMMANDABLE: 'ok',
+};
+
+export const hexapodCompensationModeStateMap = {
+  false: 'DISABLED',
+  true: 'ENABLED',
+};
+
+export const hexapodCompensationModeStatetoStyle = {
+  DISABLED: 'undefined',
+  ENABLED: 'ok',
+};
+
+export const hexapodInterlockStateMap = {
+  Disengaged: 'Disengaged',
+  Engaged: 'Engaged',
```

Comment:
If this is because the text displayed on the interface needs to be camelcase, i.e. `Engaged`, then we should review this situation, as all other state maps use uppercase states, thus they display uppercase texts on the interface.